go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/sigstore/rekor
 
-go 1.21
+go 1.22
```

Comment:
can you go mod tidy as well? i assume this will add the toolchain directive

Reply:
i did, and did not updated